robottelo.properties.sample

```diff
@@ -60,3 +60,8 @@ admin.password=changeme
 driver=firefox
 
 # NOTE: Candlepin url accepts just the hostname.
+
+# Section for declaring Sat5->Sat6 transition parameters
+[transitions]
+# Path to exported data archive (typically a .tgz containing a bunch of CSV files together with repo data
+export_tar.url=
```

Comment:
FYI, as things currently are, this will succeed:

```python
>>> from robottelo.common import conf
>>> conf.properties['main.exported_tar.url'] == ''
True
```